2020-12-24 23:47:05.025 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
2020-12-24 23:47:06.010 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
2020-12-24 23:47:07.010 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
2020-12-24 23:47:08.010 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
2020-12-24 23:48:22.027 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
2020-12-24 23:48:23.011 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
2020-12-24 23:48:24.011 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
2020-12-23 14:42:19.348 [main] ERROR o.springframework.test.context.TestContextManager - Caught exception while allowing TestExecutionListener [org.springframework.test.context.support.DependencyInjectionTestExecutionListener@7f1302d6] to prepare test instance [com.ex.evil_log.springaop.TestDemo2@37d4349f]
java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:132)
	at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:123)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:118)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:83)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:244)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:227)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:289)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:291)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:246)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:97)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:190)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:47)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:242)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'StudentDaoProxy' defined in class path resource [resources.xml]: Initialization of bean failed; nested exception is org.springframework.beans.TypeMismatchException: Failed to convert property value of type 'java.lang.String' to required type 'java.lang.Class[]' for property 'proxyInterfaces'; nested exception is java.lang.IllegalArgumentException: Could not find class [com.imooc.aop.demo1.StudentDao]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:617)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:531)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:925)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:923)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:588)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:127)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:60)
	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.delegateLoading(AbstractDelegatingSmartContextLoader.java:275)
	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.loadContext(AbstractDelegatingSmartContextLoader.java:243)
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99)
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124)
	... 25 common frames omitted
Caused by: org.springframework.beans.TypeMismatchException: Failed to convert property value of type 'java.lang.String' to required type 'java.lang.Class[]' for property 'proxyInterfaces'; nested exception is java.lang.IllegalArgumentException: Could not find class [com.imooc.aop.demo1.StudentDao]
	at org.springframework.beans.AbstractNestablePropertyAccessor.convertIfNecessary(AbstractNestablePropertyAccessor.java:598)
	at org.springframework.beans.AbstractNestablePropertyAccessor.convertForProperty(AbstractNestablePropertyAccessor.java:607)
	at org.springframework.beans.BeanWrapperImpl.convertForProperty(BeanWrapperImpl.java:219)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.convertForProperty(AbstractAutowireCapableBeanFactory.java:1740)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyPropertyValues(AbstractAutowireCapableBeanFactory.java:1696)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1436)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:608)
	... 39 common frames omitted
Caused by: java.lang.IllegalArgumentException: Could not find class [com.imooc.aop.demo1.StudentDao]
	at org.springframework.util.ClassUtils.resolveClassName(ClassUtils.java:334)
	at org.springframework.beans.propertyeditors.ClassArrayEditor.setAsText(ClassArrayEditor.java:71)
	at org.springframework.beans.TypeConverterDelegate.doConvertTextValue(TypeConverterDelegate.java:429)
	at org.springframework.beans.TypeConverterDelegate.doConvertValue(TypeConverterDelegate.java:402)
	at org.springframework.beans.TypeConverterDelegate.convertIfNecessary(TypeConverterDelegate.java:155)
	at org.springframework.beans.AbstractNestablePropertyAccessor.convertIfNecessary(AbstractNestablePropertyAccessor.java:588)
	... 45 common frames omitted
Caused by: java.lang.ClassNotFoundException: com.imooc.aop.demo1.StudentDao
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at org.springframework.util.ClassUtils.forName(ClassUtils.java:284)
	at org.springframework.util.ClassUtils.resolveClassName(ClassUtils.java:324)
	... 50 common frames omitted
2020-12-23 16:56:03.018 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
2020-12-23 16:56:04.018 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
2020-12-23 16:56:05.065 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
2020-12-23 16:56:06.003 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
2020-12-23 16:56:07.019 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
2020-12-23 16:56:08.003 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
2020-12-23 16:56:09.003 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
2020-12-23 16:56:10.004 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
2020-12-23 16:56:11.003 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
2020-12-23 16:56:12.004 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
2020-12-23 16:56:13.004 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
2020-12-23 16:56:14.004 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
2020-12-23 16:56:15.004 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
2020-12-23 16:56:16.004 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
2020-12-23 16:56:17.004 [pool-2-thread-1] ERROR com.ex.evil_log.logutil.AnnotationTask - ======error
